.gitignore

```diff
@@ -1,2 +1,3 @@
 .env
 node_modules
+.DS_Store
```

Comment:
We still need to remove the file from the branch 👍

Reply:
The `.DS_Store` file is still committed, let's make sure we remove it.

Reply:
I did not understand what you wanted and moved it to another place. In principle, I have already removed the DS_Store from the two .gitignore files.

Reply:
Sorry, I might not have explained myself. I think it is good to have them in the `.gitignore`. What I meant is that a `.DS_Store` file was actually pushed in this PR (see [here](https://github.com/DanielMoraDC/festwrap/pull/4/files#diff-e2eb93a61ffd7877ea5c751abcb3a618e8e2e9a2073a27f66d4114fe10819f86)). We should remove it.